addons/website_sale_delivery/controllers/main.py

```diff
@@ -193,17 +193,26 @@ def express_checkout_process_shipping_address(self, partial_shipping_address):
                     name=_('Anonymous express checkout partner for order %s', order_sudo.name),
                 )
 
-        # Returns the list of develivery carrier available for the sale order.
-        return sorted([{
-            'id': carrier.id,
-            'name': carrier.name,
-            'description': carrier.website_description,
-            'minorAmount': payment_utils.to_minor_currency_units(
-                WebsiteSaleDelivery._get_rate(carrier, order_sudo, is_express_checkout_flow=True)['price'],
-                order_sudo.currency_id,
-            ),
-        } for carrier in order_sudo._get_delivery_methods()],
-        key=lambda carrier: carrier['minorAmount'])
+        # Returns the list of delivery carrier available for the sale order.
+        return sorted(
+            [{
```

Comment:
Avoid doing cosmetics changes in stable
```suggestion
        return sorted([{
```